util/src/main/java/org/killbill/billing/util/export/dao/DatabaseExportDao.java

```diff
@@ -94,21 +95,32 @@ public void exportDataForAccount(final DatabaseExportOutputStream out, final Int
         }
 
         final List<ColumnInfo> columnsForTable = new ArrayList<ColumnInfo>();
+        // Separate lookup table, to keep the ordering of the columns
+        final Map<String, Integer> columnsLookup = new TreeMap<>(String.CASE_INSENSITIVE_ORDER); // Ignore casing (for H2)
```

Comment:
Why do we need to sort the keys -> isn't the point of the value (the index) to specify which column comes first?

Reply:
We actually don't need the ordering, we just need a map with case-insensitive keys.

Reply:
Right, so technically you don't need a `TreeMap` (but not that it matters)